runtime-image/jdk8-overrides.yaml

```diff
@@ -31,8 +31,14 @@ packages:
           - python-requests
 
 osbs:
-      configuration:
-            container_file: container.yaml
-      repository:
-            name: containers/jboss-eap-74-openjdk8-runtime
-            branch: jb-eap-7.4-rhel-7
+  configuration:
```

Comment:
idem?

Reply:
Same as above. We could make seperate container.yaml files for multi / non-multi, but I think it makes more sense to me to just make the expected values explicit in each descriptor / override.